ui/src/organizations/components/UpdateBucketOverlay.tsx

```diff
@@ -75,7 +75,7 @@ export default class BucketOverlay extends PureComponent<Props, State> {
     )
 
     if (!rule) {
-      return 3600
+      return 0
```

Comment:
This could probably be an extracted const as well

Reply:
Okay, I will add it to exported const :)